content/applications/hr/payroll/payslips.rst

```diff
@@ -190,22 +211,20 @@ Click on an individual payslip to view the details for that payslip on a separat
 breadcrumb menu, click :guilabel:`Employee Payslips` to go back to the list view of all payslips.
 
 A new payslip can be created from the :guilabel:`Employee Payslips` page, by clicking the
-:guilabel:`Create` button in the upper-left corner. Doing so reveals a separate blank payslip form
+:guilabel:`New` button in the upper-left corner. Doing so reveals a separate blank payslip form
 page. On that blank payslip form page, enter all the necessary information, as described in the
-:ref:`Create a new payslip <payroll/new-payslip>` section.
+:ref:`Create new payslip <payroll/new-payslip>` section.
 
 To print PDF versions of payslips from the :guilabel:`Payslips to Pay` or :guilabel:`Employee
-Payslips` pages, first select the desired payslips by clicking on the individual checkbox next to
-each payslip to be printed. Or, click the box next to :guilabel:`Reference`, which selects all
-visible payslips on the page. Then, click the :guilabel:`Print` button to print the payslips.
-
-.. image:: payslips/print.png
-   :align: center
-   :alt: Click on the Print smart button to print payslips to a PDF.
+Payslips` pages, first select the desired payslips by clicking on the individual checkbox to the
+left of each payslip to be printed. Or, click the box to the left of :guilabel:`Reference`, which
+selects all visible payslips on the page. Then, click the :guilabel:`Print` button to print the
+payslips.
 
 Payslips can also be exported to an Excel spreadsheet. When exporting, all payslips are exported
-regardless of whether some are selected or not. Click on the :guilabel:`Export All` button (download
-icon) to export all payslips to an Excel spreadsheet.
+regardless of whether some are selected or not. Click on the :guilabel:`⚙️ (gear)` icon at the end
+of the words `Employee Payslips` in the top-left corner. This reveals a drop-down menu. Click
+:guilabel:`Export All` to export all payslips to a spreadsheet.
```

Comment:
```suggestion
regardless of whether some are selected or not. To export payslips, click on the :guilabel:`⚙️ (gear)` icon at the end
of the words `Employee Payslips` in the top-left corner. This reveals a drop-down menu. Click
:guilabel:`Export All` to export all payslips to a spreadsheet.
```
when certain payslips are selected, the actions gear icon appears at the top-center of the screen, and isn't referred to as 'Export' all...it's just 'Export' -- it may be beneficial to highlight this potential difference, as well

Reply:
I missed that- I added both options, thank you!